src/vs/workbench/browser/parts/editor/editorGroupView.ts

```diff
@@ -90,6 +90,9 @@ export class EditorGroupView extends Themable implements IEditorGroupView {
 	private readonly _onDidGroupChange = this._register(new Emitter<IGroupChangeEvent>());
 	readonly onDidGroupChange = this._onDidGroupChange.event;
 
+	private readonly _onDidModelChange = this._register(new Emitter<IGroupChangeEvent>());
```

Comment:
I would have not expected a `onDidModelChange` event on the view component, rather `view.model.onDidModelChange`. To not expose the entire `model` to clients, we can maybe return a reduced interface for this?

Reply:
Ok, I will switch it. It felt weird exposing the model to the outside world so I was trying to avoid it

Reply:
Agree it is not ideal, but it avoids having 2 events that aggregate changes where a consumer might not really know what to subscribe to?

Reply:
I wonder what is the case where the view events are different than the model events. We might be able to get rid of the view aggregate event.

Reply:
Yeah one the way we can probably figure out those events that are identical and start to deprecate/replace those to use the model event.